plotly/plotly/plotly.py

```diff
@@ -1738,7 +1554,7 @@ def icreate_animations(figure, filename=None, sharing='public', auto_open=False)
     This function is based off `plotly.plotly.iplot`. See `plotly.plotly.
     create_animations` Doc String for param descriptions.
     """
-    # Still needs doing: create a wrapper for iplot and icreate_animations
+    # TODO: create a wrapper for iplot and icreate_animations
```

Comment:
I don't think that this comment still applies anymore: https://github.com/plotly/plotly.py/blob/05328c3db09f353e92d4b04785d2b4e763421e8b/plotly/plotly/plotly.py#L1734